frontend/pulse3d/src/components/basicWidgets/DropDownWidget.js

```diff
@@ -77,17 +84,29 @@ export default function DropDownWidget({
           if (selected.length === 0) {
             return <Placeholder>{label}</Placeholder>;
           }
-          return options[selected]
+          return options[selected];
         }}
       >
-        <MenuItem disabled value="">
+        <MenuItem disabled>
           <Placeholder>{label}</Placeholder>
         </MenuItem>
-        {options.map((item, idx) => (
-          <ListItem key={idx} value={idx}>
-            {item}
-          </ListItem>
-        ))}
+        {options.map((item, idx) => {
+          return disableOptions[idx] ? (
+            <Tooltip
+              key={idx}
+              title={<TooltipText>{optionsTooltipText[idx]}</TooltipText>}
+              value={idx}
+            >
+              <div>
+                <ListItem disabled={true}>{item}</ListItem>
+              </div>
+            </Tooltip>
+          ) : (
+            <ListItem key={idx} value={idx}>
+              {item}
+            </ListItem>
+          );
+        })}
```

Comment:
Hamed thought it would be better for the list items in 'Actions' to be disabled instead of the dropdown itself  so that users know what actions are available. Then each action has a tooltip that says they must select from the table below before they become available. No tooltip when actions are enabled.